core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.html

```diff
@@ -28,6 +28,9 @@
   <form class="form-horizontal" role="form" ng-submit="updateExplorationId(explorationId)">
     <input class="protractor-test-exploration-id-input" type="text" ng-model="explorationId" ng-change="checkCanSaveExpId()">
     <button type="submit" class="protractor-test-exploration-id-save-button btn btn-success btn-sm" ng-disabled="!canSaveExpId || (explorationId === currentExplorationId)">Save</button>
+    <span class="form-text" style="font-size: smaller; color: red;" ng-if="!expIdIsValid">
```

Comment:
Please move the style to a class (instead of av inline css.)

Reply:
@DubeySandeep Done!